src/components/collectible-icon/utils/get-broken-image-size.ts

```diff
@@ -0,0 +1,4 @@
+export const getBrokenImageSize = (isBigSize: boolean) => ({
+  width: isBigSize ? 72 : 38,
```

Comment:
maybe we need to add `formatSize()` ? is it works properly on different devices?